dbml-homepage/docs/docs/README.md

```diff
@@ -160,7 +160,7 @@ Example,
         id integer [primary key]
         username varchar(255) [not null, unique]
         full_name varchar(255) [not null]
-        gender varchar(1) [default: 'm']
+        source varchar(255) [default: 'direct']
```

Comment:
Hi Raphael,

Thank you for your PR.
We believe that the we are trying to add a case of `varchar(1)` in this example. How about my suggestion?

```
gender varchar(1) [not null]
source varchar(255) [default: 'direct']
```